packages/hooks/src/useCurrencyExchangeRate.ts

```diff
@@ -0,0 +1,14 @@
+import { useStore } from '@deriv/stores';
+
+/**
+ * we can use this hook to get the exchange rate for the given currency.
+ * exchange_rates comes from store and includes the rates for all currencies based on USD.
+ */
+const useCurrencyExchangeRate = (currency: string) => {
+    const { exchange_rates } = useStore();
+    const rate = exchange_rates.data?.rates?.[currency] || 1;
+
+    return rate;
+};
+
+export default useCurrencyExchangeRate;
```

Comment:
```suggestion
const useExchangeRate = () => {
    const { exchange_rates } = useStore();
    const data = exchange_rates.data;
    const rates = data?.rates;

    const getRate = useCallback((currency: string) => rates?.[currency] || 1, [rates]);

    return {
        getRate,
        last_update: data?.date,
        base_currency: data?.base_currency || 'USD',
    };
};

export default useExchangeRate;
```

@shayan-deriv Please rename the file too and check if the documentation needs to be updated or not 🙏🏻